docs/content/contributing/terraform/testing.md

```diff
@@ -88,8 +88,9 @@ If you get failures, you should examine them to understand how you can make your
 
 ### Creating exceptions
 
-In some circumstances, you may need to create an exception for a policy, you can do so by creating a `.rego` file in the `exceptions` sub-directory of your example.
-For example, to exclude the rule called `"configure_aks_default_node_pool_zones"`, create a file called `exceptions/exception.rego` in your example, with the following content:
+As a general rule, exceptions should only be applied to module examples where you are testing valid but non-compliant configurations. Root module defaults should still comply with WAF and Security best practices. However, if you need to create an exception for a policy, you can do so by creating a `.rego` file in the `exceptions` sub-directory of your example.
+
+For example, to exclude the Azure Proactive Resiliency Library policy rule called `"configure_aks_default_node_pool_zones"`, create a file called `exceptions/exception.rego` in your example with the following content:
```

Comment:
The users might need to ignore rules in different packages, could we use package name like `aprl.rego` and `avmsec.rego` in the file name?